docs/operations/operations-import-ALZAccount.md

```diff
@@ -0,0 +1,206 @@
+# Operations - How to migrate an Amazon Landing Zone (ALZ) account "as is" into an Amazon Secure Environment Accelerator.
+
+<!-- TOC depthFrom:1 depthTo:4 -->
+
+- [1. Prerequisites (Setup)](#1-prerequisites-setup)
+- [2. Landing Zone - Disassociate the account from the Landing Zone](#2-landing-zone---disassociate-the-account-from-the-landing-zone)
+- [3. Landing Zone - Remove the account from the landing Zone (ALZ) organizations and make standalone](#3-landing-zone---remove-the-account-from-the-landing-zone-alz-organizations-and-make-standalone)
+- [4. Accelerator (ASEA) - Invite the account into it's organization](#4-acceleartor-asea---invite-the-account-into-its-organization)
```

Comment:
it's to its

Reply:
I realize markdown is tough, but could you cut-paste this into Word - run through spell and grammar checker and migrate changes back here?  (would make it quicker to review/cleanup)

Reply:
Done - made the changes  and had one commit with spelling, grammar, added some more subtitles and tried to simplify where possible.   Please feel free to edit changes directly.